README.md

```diff
@@ -7,10 +7,6 @@ Dapr is a portable, event-driven, runtime for building distributed applications
 [![Gitter](https://badges.gitter.im/Dapr/community.svg)](https://gitter.im/Dapr/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-- [dapr.io](https://dapr.io)
```

Comment:
This was intentional, this should not be removed.

Reply:
thank you! I suggest that add the url to the description of repo, ok?

Reply:
Yes, that would be great.